cnbinfo/readme.rst

```diff
@@ -0,0 +1,17 @@
+Description
```

Comment:
@jorgebalsa Have you had a look at how the README.rst looks like? If you click on 'View file', you can have a look at it and the layout doesn't look particularly good. Can you please improve it a bit by highlighting titles, define the code as code, etc?

Reply:
@adrianq 
I think I need to add a text editor to github. Do I? Which one do you recomend?
The text editor I have don't let me improve any style.